packages/webapp/src/delegates/api/eden-contract.ts

```diff
@@ -69,3 +76,28 @@ export const getMyDelegation = async (
 
     return myDelegates;
 };
+
+// eosio secondary indexes for distaccount table defined at:
+// /contracts/eden/include/distributions.hpp
+const INDEX_BY_OWNER = 2;
+
+export const getDistributionsForAccount = async (
+    account: string
+): Promise<DistributionAccount[]> => {
+    const { lower, upper } = i128BoundsForAccount(account);
+
+    if (devUseFixtureData) {
+        return DISTRIBUTION_ACCOUNTS;
+    }
+
+    const distributionRows = await getTableIndexRows(
+        CONTRACT_DISTRIBUTION_ACCOUNTS_TABLE,
+        INDEX_BY_OWNER,
+        "i128",
```

Comment:
Yeah, and this can now use TABLE_INDEXES as used in elections/api/eden-contracts.ts L120